aten/src/ATen/native/native_functions.yaml

```diff
@@ -703,13 +703,13 @@
     CPU: resize_cpu_
     CUDA: resize_cuda_
 
-- func: empty(int[] size, *, Tensor(a!) out) -> Tensor(a!)
+- func: empty(int[] size, *, MemoryFormat? memory_format=contiguous_format, Tensor(a!) out) -> Tensor(a!)
```

Comment:
Right, so it's kind of weird that you explicitly added `MemoryFormat? memory_format=contiguous_format` here and then assert that it must not have a value (even more puzzlingly, the default here is for `MemoryFormat` to be `contiguous_format` which seems to imply that it would always error by default?)

At the *very* least this deserves a comment saying why you have to do it this way, but I can't think of a good reason why you have to do it this way...

Reply:
Unfortunately codegen requires out function to have exactly same arguments list as non-out function.